README.md

```diff
@@ -30,25 +30,49 @@ to
 [announce@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/announce).
 See all the [mailing lists](https://www.tensorflow.org/community/forums).
 
-**Tensorflow ROCm port**
-This project is based on TensorFlow 1.14.0. It has been verified to work with the latest ROCm2.6 release.
-Please follow the instructions [here](https://github.com/RadeonOpenCompute/ROCm-docker/blob/master/quick-start.md) to set up your ROCm stack.
-A docker container: **rocm/tensorflow:latest(https://hub.docker.com/r/rocm/tensorflow/)** is readily available to be used:
```

Comment:
Since this points to the specific container, maybe use the tags page instead https://hub.docker.com/r/rocm/tensorflow/tags